sbnanaobj/StandardRecord/SRTrackCalo.h

```diff
@@ -21,10 +21,15 @@ namespace caf
       float dedx; //!< dQ/dx [ADC/cm] -- pre calibration and electron lifetime correction
       float pitch; //!< Track pitch [cm]
       float t; //!< Time of deposition [ticks]
-      SRVector3D p; //!< Position of deposition [cm]      
+      // NOTE: flatten-caf seems to have trouble with nesting the point
+      // inside a SRVector3D -- so just expose x/y/z out here
+      float x; //!< X-Position of deposition [cm]
+      float y; //!< Y-Position of deposition [cm]
+      float z; //!< Z-Position of deposition [cm]
       float integral; //!< Hit Charge Integral [ADC]
       float sumadc; //!< Hit Charge SummedADC [ADC]
       short wire; //!< Wire of Calo-Point
+      short tpc; //!< TPC of Calo-Point
```

Comment:
Do we want the Cryostat number as well or maybe that's superfluous since we should be able to just take the x value itself.

Reply:
The Cryostat number is in the track. I'd want to keep it out because I want to keep this object as small as possible.

Reply:
Ok yeah if it's in the track, then this sounds good to me.